lava-v2-jobs-from-api.py

```diff
@@ -222,7 +230,6 @@ def main(args):
                                         if 'nfs' in plan:
                                             nfsrootfs_url = NFSROOTFS_URL.format(initrd_arch)
                                             initrd_url = None
-                                            platform = platform + "_rootfs:nfs"
```

Comment:
Are the backend changes live to make this show as a separate boot result? otherwise this hack is still needed

Reply:
The backend changes are currently stalled as we're discussing things with @MiloCasagrande.  It's essentially the same issue as with any test that boots the same build as a plain boot, and overwrites the boot data.

So the short answer is no, there is still a known issue in the backend but it's not just about NFS rootfs so this hack doesn't really solve much.  Running the "simple" test plan will typically cause the same issue...  The backend changes under review should resolve this issue as only plain "boot" tests will then be creating a boot entry:

    https://github.com/kernelci/kernelci-backend/pull/45

Reply:
So do we...
a) hold off merging this until the backend is deployed
b) remove the simple test plan and add the _rootfs:nfs hack back
c) just deploy it as it is

Reply:
I would say, wait until we've got an idea when the backend changes will be merged and deployed.  If it's a matter of a few of days, we can hold off until then.  Otherwise, we'll see what's the best alternative based on the outcome of the backend discussion.